sdk/identity/azure-identity/azure/identity/version.py

```diff
@@ -3,4 +3,4 @@
 # Licensed under the MIT License. See LICENSE.txt in the project root for
 # license information.
 # --------------------------------------------------------------------------
-VERSION = "0.0.1"
+VERSION = "1.0.0"
```

Comment:
This should be a prerelease so 1.0.0b1 possibly?

Reply:
Yes. We should mark this as a preview release...